app/cert.pem

```diff
@@ -0,0 +1,35 @@
+-----BEGIN CERTIFICATE-----
+MIIGCzCCA/OgAwIBAgIUUQhsXysbsqfJ76nblXaXror6uNEwDQYJKoZIhvcNAQEL
+BQAwgZQxCzAJBgNVBAYTAklOMQ4wDAYDVQQIDAVJbmRpYTESMBAGA1UEBwwJQmVu
+Z2FsdXJ1MREwDwYDVQQKDAhRdWFsY29tbTEPMA0GA1UECwwGUXJhaXVtMRgwFgYD
+VQQDDA9IaW1hbnNodSBVcHJldGkxIzAhBgkqhkiG9w0BCQEWFGh1cHJldGlAcXVh
+bGNvbW0uY29tMB4XDTIzMTEzMDEwMzAyNloXDTI0MTEyOTEwMzAyNlowgZQxCzAJ
+BgNVBAYTAklOMQ4wDAYDVQQIDAVJbmRpYTESMBAGA1UEBwwJQmVuZ2FsdXJ1MREw
+DwYDVQQKDAhRdWFsY29tbTEPMA0GA1UECwwGUXJhaXVtMRgwFgYDVQQDDA9IaW1h
+bnNodSBVcHJldGkxIzAhBgkqhkiG9w0BCQEWFGh1cHJldGlAcXVhbGNvbW0uY29t
+MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAy8Saa6I4Rulg//8bBbRP
+S+YdO9X/kTO4X3Zdg+Mzl69vw+fwsLB0caOAh6P2sQZ+Vrj7cYLh01A1WNEOySvI
+P1z2xxDf/8L5VcfGF7/V5K5+yGL8Jf7b2G5Fn1Z1lKHgijYluHWZvJeEW05tc8Ti
+WHSTYX1YTHmJ9TtvjZES5qCDrpXbZpcOnW7qbxehRSrzaaIaKWEevFcQXVUhA5wI
+JbfZs/kXu5eGxzqg95YyjHlLk2lbGQYz7hlkjgM5D4ekALCbqLWFuNP6QqsSC/bi
+tS2Sim1NVRwFuWgTa5V0t53RvNZkN75m1bPlQkOW4ROlifP9zsRHQA09UaHpRqUC
+VueqorrOUOkpWcZ3e0YuPWAk/4xljZI4iw75XaSwiZ/BrUXDCNnHr+KLaUuI+AOI
+9io2njAp7aDx8zLIU9m7fdTBnN6qvfCsL5zoI2hfzOrz+Ir8ValruwYmWH/hV/0P
+xaT+zufZTqFEQsPCgXp4z0r+fVeUYwcAUnjD65/IPXbYvs/42LEkVZZjamPdohZk
+nmiOObbX+a8637ANq8FRdn6VlpXNdkq1cWGn65xXSNbfFG6xcIV7w+oH03jEyWx1
+wtT9Rr+xVlf9vTgO2Ao9UoKpImVU9wiXmRQLZeGrez8S8YVJhLN6HaKIV3S9uS7U
+ug0DpsSLpd1xDhGwzLEpDRUCAwEAAaNTMFEwHQYDVR0OBBYEFJoVccLyUC8VHude
+akBh7kNF9qX/MB8GA1UdIwQYMBaAFJoVccLyUC8VHudeakBh7kNF9qX/MA8GA1Ud
```

Comment:
is it okay to put this `cert.pem` here,? we can avoid this.

Reply:
Sure, will remove this and add instructions in readme to generate them